keyboards/gggw/crosses/info.json

```diff
@@ -0,0 +1,28 @@
+{
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "oled": true,
```

Comment:
If the OLED feature is enabled at the keyboard-level, there also needs to exist some keyboard-level configuration for them in the `crosses.c` file using the `oled_task_kb()` function.